src/tabcenter.css

```diff
@@ -489,9 +501,21 @@ img, .tab *:not(.clickable) {
 /* DARK THEME CUSTOMIZATIONS */
 body.dark-theme {
     --tab-background-normal: 223, 15.2%, 18%;
-    --tab-background-active: 221, 41.4%, 53.1%;
-    --tab-background-hover: 222, 28.3%, 35.55%;
-    color: rgb(245, 247, 250);
+    --tab-background-active: 221, 41.4%, 33.1%;
+    --tab-background-hover: 222, 28.3%, 25.55%;
+    color: #c0c0c0; /*Lightened text to make it easier to read*/
+}
+
+/*Unselected tab text is slightly muted in appearance*/
+body.dark-theme .tab{
+  color: #808080;
+}
+
+/*Adds a background to tab icons to make them easier to see*/
+body.dark-theme #tablist.shrinked .tab-icon{
+  background-color: #e6e6e6;
+  border-radius: 20px;
+  filter: brightness(50%);
```

Comment:
Let's keep it vanilla and drop these, I suggest we wait and see what Mozilla does with the black tab bar on OSX (they will have the same "favicon brightness" problem).